platform/security/sts/security-sts-ldapclaimshandler/src/main/java/ddf/security/sts/claimsHandler/ClaimsHandlerManager.java

```diff
@@ -128,6 +134,13 @@ public void update(Map<String, Object> props) {
                 ClaimsHandlerManager.MEMBER_USER_ATTRIBUTE);
         String propertyFileLocation = (String) props.get(
                 ClaimsHandlerManager.PROPERTY_FILE_LOCATION);
+        String bindMethod = (String) props.get(ClaimsHandlerManager.BIND_METHOD);
+        String realm = (props.get(ClaimsHandlerManager.REALM) != null) ?
```

Comment:
Would it make sense to only get the `realm` and `kdcAddress` if the `bindMethod` had been determined to be GSSAPI SASL and debug log otherwise?

And would empty values for those properties be an error with GSSAPI SASL? If it couldn't find kerberos it seems that it would fail to create the binding.